packages/base/src/StaticArea.js

```diff
@@ -0,0 +1,97 @@
+let staticAreaIndex = 1;
+class StaticAreaItem extends HTMLElement {
+	constructor() {
+		super();
+	}
+}
+class StaticAreaElement extends HTMLElement {
```

Comment:
In StaticArea.js (static singleton class) remains everything that is static and not related to a particular item
- next index variable
- getInstance function